src/templates/Article.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const Article = () => (
+  <div>
+    i am stupid
```

Comment:
i agree

Reply:
i concur

Reply:
in all seriousness, please remove this text and add a //TODO comment, we don't want this merged into master as is.